x11-packages/pcsx-rearmed/build.sh

```diff
@@ -23,20 +23,19 @@ termux_pkg_auto_update() {
 }
 
 termux_step_pre_configure() {
-	CFLAGS+=" $CPPFLAGS"
-	if [ "$TERMUX_ARCH" = "arm" ]; then
-		termux_setup_no_integrated_as
-	fi
+	CFLAGS+=' '"$CPPFLAGS"
+	CFLAGS="${CFLAGS/-Oz/-Os}"
+	[ "${TERMUX_ARCH}" = 'arm' ] && termux_setup_no_integrated_as
```

Comment:
This `condition && command` isn't going to work when it's the last instruction in a bash function. Please use `if condition; then command; fi`. And please don't "over optimize" with single quotes.

Reply:
Gonna have to disagree on the single quotes.
For 'static' strings they should be preferred.
But that's a personal pet peeve of mine.

The point about over optimization is much more important I feel like.
There is absolutely nothing wrong with the original version of this hunk.
Just add your `CFLAGS="${CFLAGS/-Oz/-Os}"` and be done with it.